src/Tickets/Seating/app/blockEditor/hook-callbacks.js

```diff
@@ -2,6 +2,7 @@ import { storeName } from './store';
 import { select, dispatch } from '@wordpress/data';
 import SeatType from './header/seat-type';
 import LayoutSelect from './settings/layoutSelect';
+import UpSell from './settings/upSell';
```

Comment:
You can call the file `upsell`, no capital `S` in `sell`, since "upsell" is a legit English word.